openlibrary/plugins/openlibrary/js/jquery.customFade.js

```diff
@@ -4,8 +4,6 @@ export default function addFadeInFunctionsTojQuery($) {
     // books/edit/edition.html, publishers/view.html and subjects.html
```

Comment:
Future PR: We can now delete this entire file and use the stock jquery fadein/fadeout!

Reply:
ohhh yeh. nice!

Reply:
https://github.com/internetarchive/openlibrary/issues/5213